processor-api/src/main/java/com/synopsys/integration/alert/processor/api/filter/FilterableNotificationExtractor.java

```diff
@@ -22,10 +22,12 @@
  */
 package com.synopsys.integration.alert.processor.api.filter;
 
+import java.util.Optional;
+
 import com.synopsys.integration.alert.common.rest.model.AlertNotificationModel;
 import com.synopsys.integration.alert.processor.api.filter.model.FilterableNotificationWrapper;
 
 public interface FilterableNotificationExtractor {
```

Comment:
I think we decided we don't need an interface when there will only be one implementation, so this class can be deleted.